client/benches/torii.rs

```diff
@@ -148,14 +145,11 @@ fn instruction_submits(criterion: &mut Criterion) {
     ));
     let account_name = "account";
     let account_id = AccountId::new(account_name, domain_name).expect("Valid");
-    let create_account = RegisterBox::new(IdentifiableBox::NewAccount(
-        NewAccount::with_signatory(
-            account_id.clone(),
-            KeyPair::generate()
-                .expect("Failed to generate Key-pair.")
-                .public_key,
-        )
-        .into(),
+    let create_account = RegisterBox::new(Account::with_signatory(
```

Comment:
Still a little verbose, but IMO much better. Once we have GATs, we should probably have a generic `register` function that returns a `RegisterBox`.